### ISSUE TITLE
backticks overlaping other characters "Red Hat Mono" and "Red Hat Display"

### ISSUE THREAD

**Issue by KunalTanwar** [RedHatOfficial/RedHatFont#43]
VS Code
![image](https://user-images.githubusercontent.com/77537933/156412828-6158658c-87bf-4b48-9ca3-c44404e54bfb.png)

In Notepad
![image](https://user-images.githubusercontent.com/77537933/156412972-cac36518-ec1c-47f7-8f50-0a459f6ea134.png)

In MS Word
![image](https://user-images.githubusercontent.com/77537933/156413056-8412d8dd-daf7-4c31-8e3b-541793ee06f7.png)

In Chrome
![image](https://user-images.githubusercontent.com/77537933/156413296-16df5959-7f83-4761-b075-bb8b480db7e3.png)

Edge
![image](https://user-images.githubusercontent.com/77537933/156413494-d9784b1c-1bac-4988-968c-d02d189b7424.png)

Figma
![Group 23](https://user-images.githubusercontent.com/77537933/156414333-16af9b42-4a0a-4179-a002-32cb9a0038a6.png)

**Comment by notklaatu** [comment]
I'm also experiencing this.

Here's Red Hat Text in LibreOffice:

![Screenshot_20220503_061033](https://user-images.githubusercontent.com/2211671/166302211-6718fac3-6a75-44cc-99f3-f5d29cd85071.png)

**Comment by jeremymickel** [comment]
@KunalTanwar @notklaatu @mpieczaba

We have a fix for this, will be implementing it soon.

**Comment by wesruv** [comment]
Just ran into this on the customer portal in prod :( looking forward to the fix.

**Comment by wesruv** [comment]
@jeremymickel does this mean we can download an updated version from Google fonts?

I'm pretty sure that we just found this issue in Red Hat Text as well.

**Comment by wesruv** [comment]
This can be closed IMO

**Comment by notklaatu** [comment]
> This can be closed IMO

I disagree. This issue has  not been fixed in this repository (or at least, the release version doesn't reflect the fix.)

If the issue has been fixed, it should be merged here and released.

**Comment by mpieczaba** [comment]
> This issue has  not been fixed in this repository 

I think there is another repo (I don't know which one is canonical) and there, as far as I can remember, this issue is fixed.

**Comment by notklaatu** [comment]
I see now that updates aren't happening in this repo. It seems non-optimal for the official home of the font to not be receiving updates. I'll file a new bug about that.

**Comment by wesruv** [comment]
@notklaatu @mpieczaba sorry I didn't catch that other folks were waiting, the fix is here: https://github.com/RedHatOfficial/RedHatFont/pull/49

It's looking good to me, but I'm trying to get more formal QA on it, I think that's what we're waiting on the merge for.

Re: the two repos:
I got confused by the two repos as well, but got the story on that recently.
The typographyer's repo is upstream of this one, but we don't use the webfont from there. The font weight ends up being too light for the screen, this repo removes some of the lighter weights of the webfont so `font-weight: 400` reads well.

The typographer was able to fix this issue a while back but confusion on the two repos, but no one was clear on how to get those updates to this repo and recompile. In the MR above I think we got it.
